### PR TITLE
feat(sendgridpreview.vue): add preview parameter

### PR DESCRIPTION
**Pull request by Oluwaseunus**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/editor-ui/package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "@jsplumb/connector-bezier": "^5.13.2",
     "@jsplumb/core": "^5.13.2",
     "@jsplumb/util": "^5.13.2",
+    "@sendgrid/client": "^7.7.0",
```
Comment (GAMHnile) [inline packages/editor-ui/package.json:46]: Why did you install SendGrid client?

Comment (Oluwaseunus) [reply packages/editor-ui/package.json:46]: That's what we're using to fetch the templates.

**Comment by GAMHnile** [comment]
Node version change needed

**Comment by Oluwaseunus** [comment]
Lockfile has the same changes with node v16.20.0

**Review by GAMHnile — APPROVED** [review]

**Pull request merged**